multimodal/src/autogluon/multimodal/utils/log.py

```diff
@@ -136,3 +136,43 @@ def apply_log_filter(log_filter):
     finally:
         remove_log_filter(logging.getLogger(), log_filter)
         remove_log_filter(logging.getLogger("pytorch_lightning"), log_filter)
+
+
+def get_fit_start_message(save_path, validation_metric_name):
+    return f"""AutoMM starts to create the model. ✨
+
+- Model will be saved to "{save_path}".
+
+- Validation metric is "{validation_metric_name}".
+
+- To track the learning progress, you can open a terminal and launch Tensorboard:
+
+    ```shell
+    tensorboard --logdir {save_path}
+    ```
+
+Enjoy the coffee, and let AutoMM do the job ☕☕☕ Learn more at https://auto.gluon.ai
+"""
+
+
+def get_fit_complete_message(save_path):
+    return f"""AutoMM has created your model 🎉🎉🎉
+
+- To load the model, use the following code 
+
+    ```python
+    from autogluon.multimodal import MultiModalPredictor
+    
+    predictor = MultiModalPredictor.load("{save_path}")
+    ```
+
+- You can open a terminal and launch Tensorboard to visualize the training log:
+
+    ```shell
+    tensorboard --logdir {save_path}
```

Comment:
Would it be possible to check (without importing tensorboard) if tensorboard is available? This would be good because we can alter the message to indicate if they would need to first install tensorboard.

Reply:
I feel that Tensorboard is widely used. May be users can easily figure out how to install Tensorboard.

Reply:
sounds good